prov/efa/src/rxr/rxr.h

```diff
@@ -156,6 +156,12 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
 #define RXR_MULTI_RECV_POSTED	BIT_ULL(4)
 #define RXR_MULTI_RECV_CONSUMER	BIT_ULL(5)
 
+/*
+ * Flags to tell if the transmission is using FI_DELIVERY_COMPLETE
+ * protocols.
+ */
+#define RXR_DELIVERY_COMPLETE_REQUESTED		BIT_ULL(6)
```

Comment:
You have to go back to the 2nd commit "eager rtm delivery complete" to introduce this change.

Reply:
Currently this change is introduced in the 2nd commit. resolving.